pyiode/pyiode_ws.pyx

```diff
@@ -84,6 +84,7 @@
 
 from collections.abc import Iterable
 
+
```

Comment:
Why do you have `Merge branch 'master' into jmp_python_sync_671` ?
It sounds like you tried a git pull  but you forgot the `--rebase` argument.
To do a "clean" pull rebase, you must type:
```bash
git pull --rebase <remote> <branch>
```
If you want to clean the list of commits to push (squash, move, reword, fixup), you must type
``` bash
git pull --rebase=interactive <remote> <branch>
```

Reply:
Problem fixed